### PR TITLE
Parshvab/changes

### PR DESCRIPTION
**Pull request by parshva-b**
Swagger minor changes, small bug fixes, data changes, tables renamed

### CODE REVIEW

**Comment by parshva-b** [comment]
you might need to run docker-compose down -v to remove everything and build everything from scratch

File: orders_data.json
Comment (Gaurav2298) [inline orders_data.json:1]: Move this file inside data_files folder

**Comment by Hardik-hp** [comment]
gg

**Review by Gaurav2298 — CHANGES_REQUESTED** [review]

File: orders_data.json
Comment (Hardik-hp) [reply orders_data.json:1]: Next pr, saw this after the merge.

**Pull request merged**